apps/svelte.dev/content/blog/2025-03-01-whats-new-in-svelte-march-2025.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "What's new in Svelte: March 2025"
+description: 'Congrats to the SvelteHack winners! Plus improved SSR in Svelte and SvelteKit'
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+[SvelteHack 2024](https://hack.sveltesociety.dev/2024) has ended and the winners were announced on February 9th. Congrats to all the [winners](https://hack.sveltesociety.dev/2024/winners) who you'll also find listed in the "Community Showcase" below.
+
+But before we dive into that, let's get to some updates!
+
+## What's new in Svelte and SvelteKit
+
+- Native support for Websockets in SvelteKit is now available for testing! You can install the version of SvelteKit from the PR using pkg.pr.new and the PR number ([How To Install](https://github.com/sveltejs/kit/issues/1491#issuecomment-2645962690), [PR/Docs](https://github.com/sveltejs/kit/pull/12973))
+- `$props.id()` provides SSR-safe ID generation - unique for each instance of a component (**Svelte@5.20.0**, [Docs](https://svelte.dev/docs/svelte/props#props.id()), [#15185](https://github.com/sveltejs/svelte/pull/15185))
+- SvelteKit now supports an option for server-side route resolution. This means that instead of loading the whole routing manifest in the client, and doing the route resolution there, the server runtime is invoked for each route request (**Kit@2.17.0**, [#13379](https://github.com/sveltejs/kit/pull/13379))
+- The Vercel adapter will now generate edge function dedicated to server side route resolution when using that option in SvelteKit (**@sveltejs/adapter-vercel 5.6.0 - 5.6.2**, [#13379](https://github.com/sveltejs/kit/pull/13379), [#13409](https://github.com/sveltejs/kit/pull/13409), [#13461](https://github.com/sveltejs/kit/pull/13461))
```

Comment:
I would completely omit that - it's an internal implementation detail part of the route resolution that is mentioned just above

```suggestion
```